bot/cogs/utilities.py

```diff
@@ -52,6 +73,35 @@ def __init__(self, bot: Friendo) -> None:
         self.reminder_tasks = {}
         self.reminder_limit = 1
 
+    @staticmethod
+    def get_timedelta(a: datetime.datetime, b: datetime.datetime) -> list:
+        """Static method for getting parsed time delta between 2 datetimes."""
+        final = []
+        delta = relativedelta(a, b)
+        years = abs(delta.years)
+        months = abs(delta.months)
+        days = abs(delta.days)
+        hours = abs(delta.hours)
+        minutes = abs(delta.minutes)
+
+        if minutes and not months and not years:
+            final.append(f'{minutes} minutes')
+
+        if hours:
+            final.append(f'{hours} hours')
+
+        if days:
+            final.append(f'{days} days')
+
+        if months:
+            final.append(f'{months} months')
+
+        if years:
+            final.append(f'{years} years')
```

Comment:
I think most of this could be turned into a for loop that cheks items of a list.